extensions/amp-addthis/0.1/amp-addthis.js

```diff
@@ -286,7 +286,7 @@ class AmpAddThis extends AMP.BaseElement {
 
     this.applyFillContent(iframe);
     this.element.appendChild(iframe);
-    this.iframe_ = iframe;
+    this.iframe_ = /** @type {HTMLIFrameElement} */ (iframe);
```

Comment:
is this nullable still? can we mark it as `!`?

Reply:
`this.iframe_` is instantiated with `null`, so yes, it is nullable.

Reply:
can we make the denotation explicit by either adding `?` or `!HTMLIframeElement|null`